person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def can_use_services?
+    true if is_of_age? || @parent_permission
```

Comment:
- [ ] [OPTIONAL] Because I didn't notice this in my last review it is optional, in this line method `is_of_age?` is called, however, the method is named `of_age?`